app/services/storage_location_deactivate_service.rb

```diff
@@ -0,0 +1,21 @@
+class StorageLocationDeactivateService
+  include ServiceObjectErrorsMixin
+
+  def initialize(storage_location)
```

Comment:
```suggestion
  def initialize(storage_location:)
```
I generally like having keyword arguments so it is clear what the class is asking for.